api/openapi/api.yaml

```diff
@@ -179,3 +179,146 @@ paths:
           description: Check schematic log
       tags:
         - Model Operations
+  /storage/projects:
+    get:
+      summary: Get all storage projects the current user has access to
+      description: Gets all storage projects the current user has access to, within the scope of the 'storageFileview' attribute.
+      operationId: api.routes.get_storage_projects
+      parameters:
+        - in: query
+          name: input_token
+          schema:
+            type: string
+            nullable: false
+          description: Token
+          example: Token
+          required: true
+        - in: query
+          name: syn_master_file_view
+          schema:
+            type: string
+            nullable: false
+          description: synapse ID of master file view. 
```

Comment:
Change "synapse ID of master file view." to "ID of view listing all project data assets. E.g. for Synapse this would be the Synapse ID of the fileview listing all data assets for a given project". We treat Synapse just as an example of an asset management system.

Reply:
For this endpoint specifically, we get all storage projects the current user has access to; this endpoint doesn't require manifest filename. 

Note: if the manifest  filename is not provided, the default manifest name will be used to instantiate the storage object. For this endpoint that's fine since the manifest filename is not relevant.

Reply:
Thanks for the suggestion. I changed the description to: ID of view listing all project data assets. For example, for Synapse this would be the Synapse ID of the fileview listing all data assets for a given project.(i.e. master_fileview in config.yml)